package.json

```diff
@@ -3,67 +3,49 @@
   "private": true,
   "version": "0.29.0",
   "scripts": {
-    "customizr": "./node_modules/customizr/bin/customizr -c customizrConfig.json",
-    "images": "./node_modules/svg2png-many/bin/index.js -i public/images/svg/ -o public/images/bitmap/",
     "build:grunt:staging": "./node_modules/grunt-cli/bin/grunt --no-color build:staging",
     "build:grunt:production": "./node_modules/grunt-cli/bin/grunt --no-color build:production",
-    "build:staging": "npm run customizr && npm run build:grunt:staging && npm run images",
-    "build:production": "npm run customizr && npm run build:grunt:production && npm run images",
-    "build:container": "./node_modules/grunt-cli/bin/grunt build:container && npm run images",
-    "start": "npm run customizr && ./node_modules/grunt-cli/bin/grunt serve",
-    "test": "snyk test"
+    "build:staging": "npm run build:grunt:staging && npm run images",
+    "build:production": "npm run build:grunt:production && npm run images",
```

Comment:
Could we remove `npm run images` from this command now we have removed the command?